websites/L/LessWrong/presence.ts

```diff
@@ -21,10 +16,35 @@ presence.on("UpdateData", async () => {
 		presenceData.state = "at Homepage";
 	} else if (document.location.pathname.startsWith("/posts/")) {
 		data = document.location.pathname.split("/");
+		const postTitle =
+			document.querySelector(".PostsPageTitle-link")?.textContent ||
+			document.querySelector(".PostsPage-secondSplashPageHeader")
+				?.textContent ||
+			"Unknown Post";
 		presenceData.details = `Reading ${data[1]}`;
-		presenceData.state = `${capitalize(
-			data[3].replace(/-\d+$/, "").split("-").join(" ")
-		)}`;
+		presenceData.state = postTitle;
+
+		presenceData.buttons = [
+			{
+				label: "View Post",
+				url: new URL(document.location.pathname, window.location.origin).href,
```

Comment:
Use `document.location` for both for consistency
```suggestion
				url: new URL(document.location.pathname, document.location.origin).href,
```
Or use href directly
```suggestion
				url: document.location.href,
```